examples/v2/createPoolWeighted.ts

```diff
@@ -34,23 +35,23 @@ const createPool = async (stopForkAfterExecution = true) => {
     const signerAddress = (await client.getAddresses())[0];
     const createPool = new CreatePool();
     const poolType = PoolType.Weighted;
-    const createWeightedPoolInput: CreatePoolWeightedInput = {
+    const createWeightedPoolInput: CreatePoolV2WeightedInput = {
```

Comment:
Shows a warning for me: `'chainId' is declared here`

Reply:
Done